components/VC/Views/VCCardView.tsx

```diff
@@ -68,27 +72,60 @@ export const VCCardView: React.FC<VCItemProps> = props => {
             ? Theme.Styles.selectedBindedVc
             : Theme.Styles.closeCardBgContainer
         }>
-        <VCCardViewContent
-          vcMetadata={props.vcMetadata}
-          context={context}
-          walletBindingResponse={walletBindingResponse}
-          credential={vc}
-          verifiableCredentialData={verifiableCredentialData}
-          fields={fields}
-          wellknown={wellknown}
-          generatedOn={formattedDate}
-          selectable={props.selectable}
-          selected={props.selected}
-          service={service}
-          isPinned={props.isPinned}
-          onPress={() => props.onPress(service)}
-          isDownloading={props.isDownloading}
-          flow={props.flow}
-          isKebabPopUp={isKebabPopUp}
-          DISMISS={DISMISS}
-          KEBAB_POPUP={KEBAB_POPUP}
-          isVerified={props.vcMetadata.isVerified}
-        />
+        {props.isTopCard ? (
+          <CopilotStep
+            text={t('copilot:cardMessage')}
+            order={6}
+            name={t('copilot:cardTitle')}>
+            <CopilotView>
+              <VCCardViewContent
+                vcMetadata={props.vcMetadata}
+                context={context}
+                walletBindingResponse={walletBindingResponse}
+                credential={vc}
+                verifiableCredentialData={verifiableCredentialData}
+                fields={fields}
+                wellknown={wellknown}
+                generatedOn={formattedDate}
+                selectable={props.selectable}
+                selected={props.selected}
+                service={service}
+                isPinned={props.isPinned}
+                onPress={() => props.onPress(service)}
+                isDownloading={props.isDownloading}
+                flow={props.flow}
+                isKebabPopUp={isKebabPopUp}
+                DISMISS={DISMISS}
+                KEBAB_POPUP={KEBAB_POPUP}
+                isVerified={props.vcMetadata.isVerified}
+                isInitialLaunch={props.isInitialLaunch}
+              />
+            </CopilotView>
+          </CopilotStep>
+        ) : (
+          <VCCardViewContent
```

Comment:
Shall we extract this call to VCCardViewContent component into separate component which will call this VCCardViewContent so that whenever we need to make changes in props we will do it in one place and the chances of modifying it one place and missing it another will be reduced.

Reply:
created VCCardViewContentFn method which returns VCCardViewContent component inside this component, now chnage the props inside that method is enough